routes/User.js

```diff
@@ -0,0 +1,231 @@
+const express = require("express");
+const router = express.Router();
+const bcrypt = require("bcryptjs");
+const User = require("../db/User");
+const user_add = require("../db/address");
+const User_Login = require("../db/User_login");
+const user_verification = require("../db/vetification");
+const mongoose = require("mongoose");
+const crypto = require("crypto");
+const hash = crypto.createHash("sha256");
+const md5 = require("md5");
+const jwt = require("jsonwebtoken");
+const jwtkey = 10;
+
+require("../db/config");
+const session = require("express-session");
+router.use(express.json());
+
+router.post("/register", async (req, res) => {
+  const user = new User(req.body);
+  try {
+    const result = await user.save();
+    console.log(result);
+
+    if (result.password != result.confirm_password) {
+      return res.status(400).send("password not matched");
+    }
+
+    // const randomNum = Math.floor(Math.random() * 10000000);
+    // userID = user._id;
+
+    // req.session.randomNumber = randomNum;
+    // const salt = await bcrypt.genSalt(10);
+    const secPass = await bcrypt.hash(req.body.password, salt);
+
+    result.password = secPass;
+
+    const sec_con_Pass = await bcrypt.hash(req.body.confirm_password, salt);
```

Comment:
confirm password is only to comparing the both password, don't save it in the db